doc/source/cluster/kubernetes.rst

```diff
@@ -1,18 +1,13 @@
+.. _ray-k8s-deploy:
+
 Deploying on Kubernetes
 =======================
 
 .. note::
 
   The easiest way to run a Ray cluster is by using the built-in autoscaler,
-  which has support for running on top of Kubernetes. Please see the `autoscaler
-  documentation <autoscaling.html>`__ for details.
-
-.. warning::
-
-  Running Ray on Kubernetes is still a work in progress. If you have a
-  suggestion for how to improve this documentation or want to request a
-  missing feature, please get in touch using one of the channels in the 
-  `Questions or Issues?`_ section below.
+  which has support for running on top of Kubernetes. Please see the :ref:`autoscaler
+  documentation <ray-launch-k8s>` for details.
```

Comment:
What is the purpose of this document? You mention this reference to another doc but don't really say what the goal of this page will be.

Reply:
+1, we should mention that this is for manual control/advanced usage. More users should use cluster launcher